Day 2/DayTwoPartOne.py

```diff
@@ -0,0 +1,43 @@
+import sys
+sys.path.insert(0, "C:/Users/Callipolis/Desktop/Programming/AoC2/Advent-of-Code-2015")
+from InputReader import InputReader
+
+input_reader = InputReader("input2.txt")
+raw_input = input_reader.get_list_with_lines()
+
+class WrapperOrder:
+    
+    paper_counter = 0
+    puzzle_input = None
+    
+    def __init__(self, puzzle_input):
+        self.puzzle_input = puzzle_input
+        self.format_lines()
+
+    def line_to_iterables(self, line):
+        nums = line.split("x")
+        return nums
+    
+    def get_material_per_box(self, line):
+        length = int(line[0])
+        width = int(line[1])
+        height = int(line[2])
+        side1 = length*width
+        side2 = width*height
+        side3 = height*length
+        line_total = 2*side1 + 2*side2 + 2*side3 + min(side1, side2, side3)
+        return line_total
+    
+    def format_lines(self):
+        for line in self.puzzle_input:
+            line = self.line_to_iterables(line)
+            line_total = self.get_material_per_box(line)
+            self.paper_counter += line_total 
+    
+    def get_req_material(self):
```

Comment:
ne szokj ra a roviditesekre, sokat rontanak az olvashatosagon. te benne vagy a kontextusban amikor irod, neked egyertelmu hogy a req=required, de mas megakad, gondolkodik hogy ez mi lehet, minek a roviditese. 

Abban is hatraltat hogy fluently kiolvasda kodot a fejedben. "Get reku material"

Reply:
Full egyértelműeknél sem? Pl. character / char? Nem rontja az olvashatóságot, ha túl hosszúak a nevek?

Reply:
Hat kerdes ugye hogy mi az ami neked full egyertelmu meg mi az ami masnak is.
char az hatareset, valoszinu en is ugy irnan.
min/max/sum-ot szoktam irni minimum meg summa helyett
en meg a configurationt is kiirom amugy config helyett, de a config az pont olyan ami johet amugy
id-t irok identification helyett

en azt mondanam okol szabalynak hogy ami benne van a kozhasznalatban, amit szoban is mondanal az johet

Reply:
es akkor itt jon az "ellent mondas", de rontja a hosszu nev az olvashatosagot. de a megoldas az nem az hogy ugyanugy 8 szobol all a nev es rovidited mindet hanem az hogy mas nevet valasztasz. 
altalaban amikor tul hosszu nev kell ahhoz hogy ertheto maradjon akkor nem jo a context. lehet hianyzik egy class es tul sok minden van egy classban, lehet kene a visszateresi erteknek egy tipus, vagy a valtozo nev amiben majd az objectet/visszateresi erteket tarolod segit. itt pl line total helyett hivhatos required material-nak a valtozot es akkor a fuggveny lehet csak get material

Reply:
Köszi, azt hiszem értem. Majd jelezd kérlek, ha átlendülök és túl hosszúak lesznek.

Reply:
vagy tul absztraktak 🙂☺️😊

Reply:
💦